### ISSUE TITLE
Add support for pytorch 2.0.0

### ISSUE THREAD

**Issue by gusty1g** [deepchem/deepchem#3354]
PyTorch 2.0 have significant speed advantage on training deep neural networks and DeepChem should start supporting PyTorch 2.0. This requires torch version bumps in `requirements/torch/*`

**Comment by arunppsg** [comment]
I think not required. Going to close this issue for now.
